### PR TITLE
Bugfix/#16232 cgroup update

### PR DESCRIPTION
**Pull request by ljblancoredborder**

### CODE REVIEW

File: resources/scripts/rb_check_cgroups.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+require 'socket'
+require 'chef'
+module RedBorder
+    module Checker
+        def self.check_memservices_cgroups
+            # TODO: optimize node get with chef confifured
+            Chef::Log::info("Memservices Check")
+            # Chef::Config.from_file("/etc/chef/client.rb")
+            # Chef::Config[:client_key] = "/etc/chef/admin.pem"
+            # Chef::Config[:http_retry_count] = 5
```
Comment (malvads) [inline resources/scripts/rb_check_cgroups.rb]: why there is unused code here?

File: resources/scripts/rb_check_cgroups.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+            hostname = `hostname -s`.strip
+            # node = Chef::Node.load(hostname)
+            # memory_services=node.default.memory_services.keys()
```
Comment (malvads) [inline resources/scripts/rb_check_cgroups.rb]: also here

File: packaging/rpm/redborder-cgroups.spec [hunk 3]
```diff
@@ -41,6 +42,8 @@ systemctl enable redborder-cgroups
 %doc
 
 %changelog
+* Fri Feb 23 2024 - Luis Blanco <ljblanco@redborder.com>
+- Ruby wrapper added
```
Comment (malvads) [inline packaging/rpm/redborder-cgroups.spec]: the version is missing here

File: resources/scripts/rb_check_cgroups.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+            active_services.all? do |s|
+              cgroup = `systemctl show -p ControlGroup #{s}`.gsub('ControlGroup=','').chomp
+              s = s.delete("\",-").chomp
+              cgroup.include?("redborder-#{s}.slice") #assigned cgroup should cointain redborder-webui.slice ie, else false
+            end
+        end
+    end
+end
+
+RedBorder::Checker.check_memservices_cgroups()
```
Comment (malvads) [inline resources/scripts/rb_check_cgroups.rb]: Using parentheses with no arguments?

**Comment by malvads** [comment]
Also, this doesnt pass the rubocop linter

```
test.rb:1:1: C: [Correctable] Style/FrozenStringLiteralComment: Missing frozen string literal comment.
require 'socket'
^
test.rb:4:1: C: [Correctable] Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.
    module Checker
^^^^
test.rb:4:5: C: Style/Documentation: Missing top-level documentation comment for module RedBorder::Checker.
    module Checker
    ^^^^^^^^^^^^^^
test.rb:5:5: C: [Correctable] Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.
        def self.check_memservices_cgroups
    ^^^^
test.rb:5:9: C: Metrics/MethodLength: Method has too many lines. [11/10]
        def self.check_memservices_cgroups ...
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
test.rb:7:9: C: [Correctable] Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.
            Chef::Log::info("Memservices Check")
        ^^^^
test.rb:7:22: C: [Correctable] Style/ColonMethodCall: Do not use :: for method calls.
            Chef::Log::info("Memservices Check")
                     ^^
test.rb:7:29: C: [Correctable] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
            Chef::Log::info("Memservices Check")
                            ^^^^^^^^^^^^^^^^^^^
test.rb:14:28: C: [Correctable] Layout/SpaceAroundOperators: Surrounding space missing for operator =.
            memory_services=`knife node show #{hostname} -l -F json | jq '.default.redborder.memory_services | keys[]'`.chomp.lines
                           ^
test.rb:14:121: C: Layout/LineLength: Line is too long. [131/120]
            memory_services=`knife node show #{hostname} -l -F json | jq '.default.redborder.memory_services | keys[]'`.chomp.lines
                                                                                                                        ^^^^^^^^^^^
test.rb:15:28: C: [Correctable] Layout/SpaceAroundOperators: Surrounding space missing for operator =.
            active_services=memory_services.select do |s|
                           ^
test.rb:16:13: C: [Correctable] Layout/IndentationWidth: Use 2 (not 4) spaces for indentation.
                `systemctl is-active #{s}`.chomp == 'active'
            ^^^^
test.rb:19:82: C: [Correctable] Layout/SpaceAfterComma: Space missing after comma.
              cgroup = `systemctl show -p ControlGroup #{s}`.gsub('ControlGroup=','').chomp
                                                                                 ^
test.rb:20:28: C: [Correctable] Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
              s = s.delete("\",-").chomp
                           ^^^^^^
test.rb:21:55: C: [Correctable] Layout/LeadingCommentSpace: Missing space after #.
              cgroup.include?("redborder-#{s}.slice") #assigned cgroup should cointain redborder-webui.slice ie, else false
                                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
test.rb:21:121: C: Layout/LineLength: Line is too long. [123/120]
              cgroup.include?("redborder-#{s}.slice") #assigned cgroup should cointain redborder-webui.slice ie, else false
                                                                                                                        ^^^
test.rb:27:45: C: [Correctable] Style/MethodCallWithoutArgsParentheses: Do not use parentheses for method calls with no arguments.
RedBorder::Checker.check_memservices_cgroups()
                                            ^^

1 file inspected, 17 offenses detected, 13 offenses autocorrectable
```

**Review by malvads — DISMISSED** [review]
This PR needs to be linted before merge

**Review by ViMesTor — APPROVED** [review]

**Pull request merged**